keadm/cmd/keadm/app/cmd/util/common.go

```diff
@@ -293,7 +295,17 @@ func isK8SComponentInstalled(kubeConfig, master string) error {
 		return fmt.Errorf("Failed to get the version of K8s master, please check whether K8s was successfully installed, err: %v", err)
 	}
 
-	k8sMinorVersion, _ := strconv.Atoi(serverVersion.Minor)
+	return checkKubernetesVersion(serverVersion)
+}
+
+func checkKubernetesVersion(serverVersion *version.Info) error {
+	reg := regexp.MustCompile(`[[:digit:]]*`)
+	minorVersion := reg.FindString(serverVersion.Minor)
+
+	k8sMinorVersion, err := strconv.Atoi(minorVersion)
+	if err != nil {
+		return fmt.Errorf("Could not parse the minor version of K8s, please open an issue on https://github.com/kubeedge/kubeedge with the following error message: %s", err)
```

Comment:
```suggestion
		return fmt.Errorf("Could not parse the minor version of K8s, error: %s", err)
```
How about the log errors focusing on the error itself? :)